WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCEntities/VCEntities/presentation/PresentationResponseContainer.swift

```diff
@@ -46,22 +47,48 @@ struct PresentationResponseContainer: ResponseContaining {
         self.audienceUrl = audience
         self.audienceDid = audienceDid
         self.nonce = nonce
-        self.presentationDefinitionId = presentationRequest.content.claims?.vpToken?.presentationDefinition?.id
         self.request = presentationRequest
         self.expiryInSeconds = exp
     }
     
     init(audienceUrl: String,
-                audienceDid: String,
-                nonce: String,
-                expiryInSeconds: Int,
-                presentationDefinitionId: String)
-    {
+         audienceDid: String,
+         nonce: String,
+         expiryInSeconds: Int) {
         self.audienceDid = audienceDid
         self.audienceUrl = audienceUrl
         self.expiryInSeconds = expiryInSeconds
         self.nonce = nonce
-        self.presentationDefinitionId = presentationDefinitionId
         self.request = nil
     }
+    
+    mutating func addVerifiableCredential(id: String, vc: VerifiableCredential) throws {
```

Comment:
Is this equivalent to Requirement fulfill in Wallet Library?

Reply:
yea its a helper method that the requirment will use.